roles/confluent.control_center/tasks/main.yml

```diff
@@ -168,6 +168,7 @@
     path: /var/log/confluent/
     owner: "{{control_center_user}}"
     group: "{{control_center_group}}"
+    mode: 0770
```

Comment:
This seem a bit odd, but I think it has to correspond with the task below: `Create Logs Directory`.

Reply:
Ya theres a chance this is leftover from when we didnt properly use the `LOG_DIR` env var, matching the following task makes sense

Reply:
Maybe you should create an issue to fix this in a future release? It seems a bit strange to have different permissions that for the other component log directories.

Reply:
Ya ill create an issue, i'm pretty sure this task was added to help w the archive install when the logging logic had flaws. It should be removed, but needs to be tested